content/influxdb/v2.2/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,98 @@ menu:
 weight: 101
 ---
 
+## v2.2 [TBD]
+
+This release includes the following new [features](#features) and several [bug fixes](#bug-fixes).
+
+### Features
+
+- [Technical preview of replicating data remotely](#replicate-data-remotely-technical-preview)
+- [Flux updates](#flux-updates)
+- [Build maintenance](#build-maintenance)
+- [Task metadata](#task-metadata)
+- [Metrics](#metrics)
+- [Display runtime configuration in use by `influxd`](#display-runtime-configuration-in-use-by-influxd)
+
+#### Replicate data remotely (technical preview)
+
+- Add the option to [replicate data from InfluxDB OSS to InfluxDB Cloud](/influxdb/v2.2/write-data/replication).
+
+  In this **technical preview**, you can replicate inbound data using the HTTP API. Support for replicating data via tasks (for example, downsampling) is targeted prior to the official release of this feature.
+  This feature helps you to collect data locally from multiple InfluxDB OSS edge deployments, analyze and aggregate the data, and then send the data to one or more buckets on a remote InfluxDB Cloud instance. If InfluxDB OSS loses connectivity to InfluxDB Cloud, the data is sent when network connectivity is restored.
```

Comment:
I think this is a little bit confusing. We used to have a subscription mechanism in v1.  The difference is that the the subscription wasn't durable.  Meaning, it was fire and forget.  If the upstream connectivity was lost, the data was not stored, buffered or preserved in any way.   What we are introducing here is more of a "durable subscription."  I would propose:

Technical  Preview: Durable Subscription

The feature helps allows you to create a durable subscription on a per bucket basis for the purposes of replicating data from  InfluxDB OSS instance to InfluxDB Cloud.  This allows users to store, analyze and aggregate data locally while also forwarding newly arriving data to a centralized InfluxDB Cloud account.  If InfluxDB OSS loses connectivity to InfluxDB Cloud, the data is sent when network connectivity is restored.  Configuration options include a maximum buffer size and data age restrictions to restrict the amount of data stored on disk.

Limitations:
Currently only data received directly via the v2 write API is stored locally within the InfluxDB OSS instance and replicated.  This means that data created using Flux tasks is not replicated.  This limitation is intended to be resolved as part transitioning this feature from technical preview to production ready.  Additional performance enhancements are also planned.

If you add a durable subscription to an existing bucket, any data already present within the bucket is not replicated.

Reply:
Feedback:  
We are looking for feedback regarding this technical preview feature.  You can contact us via slack (link to community slack) or our community site (community.influxdata.com).

Reply:
added this issue: https://github.com/influxdata/docs-v2/issues/3777 to add definition of technical preview

Reply:
Borrowed Tim's wording to clarify what this durable subscription feature is.